node-graph/gcore/src/graphic_element/renderer.rs

```diff
@@ -652,10 +698,22 @@ impl GraphicElementRendered for VectorDataTable {
 					subpath
 				};
 
+				// For free-floating anchors, we need to add a click target for each
+				let single_anchors = instance.point_domain.ids().iter().filter(|&&point_id| instance.connected_count(point_id) == 0);
+				let single_anchors_targets = single_anchors
+					.map(|&point_id| {
+						let anchor = instance.point_domain.position_from_id(point_id).unwrap_or_default();
+						let group = ManipulatorGroup::new_anchor_with_id(anchor, point_id);
+
+						ClickTarget::new_with_point_group(group)
+					})
+					.collect::<Vec<ClickTarget>>();
```

Comment:
It is not ergonomic or performant to collect and then iterate. There is no need for this heap allocation.